demo/docker-compose.yaml

```diff
@@ -55,37 +55,37 @@ services:
     environment:
       SW_OAP_ADDRESS: "http://oap:12800"
 
-  zookeeper:
-    container_name: zk
-    image: confluentinc/cp-zookeeper:latest
-    ports:
-      - "2181:2181"
-    environment:
-      ZOOKEEPER_CLIENT_PORT: 2181
-      ZOOKEEPER_TICK_TIME: 2000
-    networks:
-      - manual
-
-  kafka:
-    container_name: kafka
-    image: confluentinc/cp-kafka
-    expose:
-      - 9092
-      - 9094
-    ports:
-      - 9092:9092
-      - 9094:9094
-    depends_on:
-      - zookeeper
-    environment:
-      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
-      KAFKA_LISTENERS: INTERNAL://0.0.0.0:9092,OUTSIDE://0.0.0.0:9094
-      KAFKA_ADVERTISED_LISTENERS: INTERNAL://kafka:9092,OUTSIDE://localhost:9094
-      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: INTERNAL:PLAINTEXT,OUTSIDE:PLAINTEXT
-      KAFKA_INTER_BROKER_LISTENER_NAME: INTERNAL
-      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
-    networks:
-      - manual
+#  zookeeper:
```

Comment:
And this. Please also check the other places, I didn't point out all commented places.